tests/mock_tables/dbconnector.py

```diff
@@ -133,12 +134,22 @@ def keys(self, pattern='*'):
         # Find every key that matches the pattern
         return [key for key in self.redis.keys() if regex.match(key)]
 
+def _isInit():
+    return   SonicDBConfig._sonic_db_config_init
+
+def  _initialize():
```

Comment:
>   [](http://example.com/codeflow?start=3&length=2)

Use one space instead of two.

Reply:
Removed this code as it will be added in swsssdk.